src/app/Instance/ErrorHandling/ErrorHandlingEdit.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { FormEvent, useCallback, useEffect, useState } from "react";
+import {
+  ActionGroup,
+  Alert,
+  Button,
+  Form,
+  FormAlert,
+  PageSection,
+} from "@patternfly/react-core";
+import { useTranslation } from "@rhoas/app-services-ui-components";
+import ErrorHandlingCreate from "@app/Instance/ErrorHandling/ErrorHandlingCreate";
+
+export interface ErrorHandlingEditProps {
+  apiError?: string;
+  getSchema: (method: string) => Promise<object>;
+  isLoading: boolean;
+  method?: string;
+  onCancelEditing: () => void;
+  onSubmit: (method?: string, parameters?: object, schema?: object) => void;
+  parameters?: Record<string, unknown>;
+  registerValidateParameters: (validationFunction: () => boolean) => void;
+  schema?: object;
```

Comment:
I've was about to ask if we needed a test covering the whole editing/submission of `ErrorHandlingEdit` and I've realized it has a `schema` prop that's not actually used.

Looking at the props I see `ErrorHandlingEdit` could eventually manage the form validation internally (and the `bridgeNotChanged` logic too) and only call `onSubmit` when the form is valid and the actual API call can be done. 
It could be a more self-contained approach. Just a consideration.

Reply:
Good catch for the unused component prop.

Regarding component composability, your point is perfectly valid, and I think it is about personal choices. For this case, I prefer to manage the call and the possible API error on a higher level (what I consider a "page" component) that must be covered by e2e tests.
Indeed I covered the `ErrorHandlingEdit`'s `onSubmit` callback with the `should call onSubmit callback, when save button is pressed` test case.
Also, regarding the logic about finalizing or not the API submission, my design choice was to decide elsewhere (the parent component), while `ErrorHandlingEdit` is only responsible for calling the callback.